WordPress/src/main/res/values/strings.xml

```diff
@@ -1204,7 +1204,7 @@
     <string name="editor_post_title_placeholder">Post Title</string>
     <string name="editor_page_title_placeholder">Page Title</string>
     <string name="editor_content_placeholder">Share your story here…</string>
-    <string name="editor_post_saved_online">Post saved online</string>
+    <string name="editor_post_saved_online">Post moved to drafts</string>
```

Comment:
I accidentally tested this PR with a change for `editor_draft_saved_online` because that's the one that'll be shown after a trashed post is moved to draft. I think the notes in my review are still valid.